tools/releaseBuild/Image/DockerFile

```diff
@@ -0,0 +1,34 @@
+# escape=`
+#0.3.6 (no powershell 6)
+# FROM microsoft/windowsservercore
+FROM microsoft/dotnet-framework:4.7.1

```

Comment:
@TravisEz13 @JamesWTruher Could building with .net 4.7.1 cause issues for customers that do not have this relatively new runtime installed?

Reply:
For Full DotNet, it is supposed to be compatible between major versions unless the product uses features that are specific to the newer version.  I've worked on other products that did exactly what this is doing but were just very careful not to use newer features.